.travis.yml

```diff
@@ -0,0 +1,22 @@
+matrix:
+  include:
+    - language: python
+      python:
+      - "3.6"
+      install:
+        - pip3 install -r requirements3.txt
+      script:
+        - make test
+        - make lint
+
+    - language: node_js
+      node_js:
+        - "8"
+      before_install:
+        - cd amundsen_application/static
+      install:
+        - npm install
+      script:
+        - npm run build
```

Comment:
I'm still not sure about running `npm run build` which is a script written to run tests and build the source, as opposed to explicitly running the scripts we have for explicitly running tests. Thoughts?

Reply:
As far as I can tell we will some combination of `tsc, lint, and test`. `build` is a fairly nice placeholder until we can fix `test` and `tsc`.

Reply:
Resolving, we will modify after fixing our `npm run tsc` script.